### PR TITLE
DOC: fix various typos in plotting documentation

### PR DESCRIPTION
**Pull request by partev**
- [ ] closes #xxxx (Replace xxxx with the GitHub issue number)
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

### CODE REVIEW

File: pandas/plotting/_core.py [hunk 1]
```diff
@@ -308,7 +308,7 @@ def hist_frame(
 
 See Also
 --------
-Series.plot.hist: Make a histogram.
+pandas.Series.plot.hist: Make a histogram.
```
Comment (phofl) [inline pandas/plotting/_core.py:311]: This renders properly? https://pandas.pydata.org/docs/dev/reference/api/pandas.DataFrame.boxplot.html

File: pandas/plotting/_misc.py [hunk 2]
```diff
@@ -274,12 +274,13 @@ def andrews_curves(
 
     Andrews curves have the functional form:
 
-    f(t) = x_1/sqrt(2) + x_2 sin(t) + x_3 cos(t) +
-           x_4 sin(2t) + x_5 cos(2t) + ...
+    .. math::
```
Comment (phofl) [inline pandas/plotting/_misc.py:277]: Can you show a screen shot how this renders?

File: pandas/plotting/_core.py (hunk 1 above)
Comment (partev) [reply pandas/plotting/_core.py:311]: Indeed this is rendered correctly now. It seemed to be broken yesterday. I will remove this change and resubmit the pull request.

File: pandas/plotting/_misc.py (hunk 2 above)
Comment (partev) [reply pandas/plotting/_misc.py:277]: image

File: pandas/plotting/_core.py (hunk 1 above)
Comment (phofl) [reply pandas/plotting/_core.py:311]: Ah nice catch, thx

**Review by phofl — APPROVED** [review]

**Comment by phofl** [comment]
thx @partev

**Pull request merged**